packages/eas-cli/src/submissions/utils/errors.ts

```diff
@@ -57,6 +58,10 @@ const SubmissionErrorMessages: Record<SubmissionErrorCode, string> = {
     `${learnMore('https://docs.expo.io/distribution/app-signing')}`,
   [SubmissionErrorCode.IOS_INCORRECT_CREDENTIALS]:
     'Your Apple ID or app-specific password is incorrect. Please verify that you entered them correctly and try again.',
+  [SubmissionErrorCode.IOS_IPAD_INVALID_ORIENTATION]:
+    "Your app doesn't support iPad multitasking and has to require full screen.\n" +
+    "If you're submitting a managed Expo project, set the `ios.requireFullScreen` to true in app.json and build the project again.\n" +
```

Comment:
```suggestion
    "If you're submitting a managed Expo project, set the `expo.ios.requireFullScreen` to true in app.json and build the project again.\n" +
```

consistent with `expo.ios.buildNumber` and `expo.android.versionCode` above